test/git.spec.ts

```diff
@@ -38,11 +38,21 @@ const gitHubContext = {
         repo: 'repo',
         owner: 'user',
     },
+    payload: {
+        repository: {
+            html_url: 'https://github.com',
```

Comment:
please provide a real URL for testing.
```suggestion
            html_url: 'https://github.com/benchmark-action/github-action-benchmark',
```